pgtrigger/tests/models.py

```diff
@@ -13,6 +13,56 @@ class Meta:
         unique_together = ('int_field', 'char_field')
 
 
+class LogEntry(models.Model):
+    """Created when ToLogModel is updated"""
+
+    level = models.CharField(max_length=16)
+    old_field = models.CharField(max_length=16, null=True)
+    new_field = models.CharField(max_length=16, null=True)
+
+
+@pgtrigger.register(
+    pgtrigger.Trigger(
+        level=pgtrigger.Statement,
+        operation=pgtrigger.Update,
+        when=pgtrigger.After,
+        func=f'''
+            INSERT INTO {LogEntry._meta.db_table}(level)
+            VALUES ('STATEMENT');
+            RETURN NULL;
+        ''',
+    ),
+    pgtrigger.Trigger(
+        level=pgtrigger.Statement,
+        operation=pgtrigger.Update,
+        when=pgtrigger.After,
+        referencing=pgtrigger.Referencing(old='old_values', new='new_values'),
+        func=f'''
+            INSERT INTO {LogEntry._meta.db_table}(level, old_field, new_field)
+            SELECT 'STATEMENT' AS level,
+                   old_values.field AS old_field,
+                   new_values.field AS new_field
+                 FROM old_values
+                 JOIN new_values ON old_values.id = new_values.id;
+            RETURN NULL;
+        ''',
+    ),
```

Comment:
@tomage @ryanhiebert this is a test case that defines some statement-level triggers to see what it looks like in practice. One of these statement-level triggers references transition tables too